assets/src/edit-story/output/element.js

```diff
@@ -25,43 +26,57 @@ import WithLink from '../components/link/output';
 
 function OutputElement({ element }) {
   const { id, opacity, type } = element;
-
   const { Output } = getDefinitionForType(type);
 
   // Box is calculated based on the 100%:100% basis for width and height
   const box = getBox(element, 100, 100);
   const { x, y, width, height, rotationAngle } = box;
 
   return (
-    <WithMask
-      element={element}
-      box={box}
-      id={'el-' + id}
-      className="wrapper"
+    <div
       style={{
+        position: 'absolute',
```

Comment:
There still should be `pointerEvents: none` here, right? Otherwise the div will block all pointer events in areas that are cut off by a mask.

Reply:
Yes, you're correct, it completely slipped my mind. I fixed it and I added `pointerEvents: none;` to all the additional wrappers that may get generated for layered animations.

Reply:
The reason we put `pointerEvents: none` in the top-most element is exactly to avoid putting `pointerEvents: none` in every wrapper between the topmost wrapper and the mask. They don't hurt, of course. Just not needed.

Reply:
Yeah I just googled that after I committed to see if it was needed :D

Reply:
I'm removing the extra pointerEvent declarations to keep the code clean.  Thanks for the callout and the review!